source/fundamentals/transactions.txt

```diff
@@ -123,6 +123,15 @@ If you require more control over your transactions, you can use the ``startTrans
 method. You can use this method with the ``commitTransaction()`` and ``abortTransaction()``
 methods described in the preceding section to manually manage the transaction lifecycle. 
 
+.. sharedinclude:: dbx/transactions-parallelism.rst
+
+   .. replacement:: driver-specific-content
```

Comment:
I would leave the bulk write piece out until we have docs to link to. the current docs don't include client bulk writes